DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    ice_10[0].plot()
</pre><p>let's <em>not</em> use <code>dataarray[0]</code>  but instead use <code>dataarray.sel(time=....)</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>